Modules/_testcapi/abstract.c

```diff
@@ -78,6 +78,25 @@ object_hasattrstringwitherror(PyObject *self, PyObject *args)
     RETURN_INT(PyObject_HasAttrStringWithError(obj, attr_name));
 }
 
+static PyObject *
+object_copy_to_object(PyObject *self, PyObject *args)
+{
+    PyObject *obj;
+    Py_ssize_t len;
+    int result;
+    char *buf, fort;
+
+    if (!PyArg_ParseTuple(args, "Os#c", &obj, &buf, &len, &fort)) {
```

Comment:
You should not expect Unicode, but raw bytes:

```suggestion
    if (!PyArg_ParseTuple(args, "Oy#c", &obj, &buf, &len, &fort)) {
```